account_analytic_parent/views/account_analytic_account_view.xml

```diff
@@ -12,5 +12,49 @@
                 </field>
             </field>
         </record>
+
+        <record id="view_account_analytic_account_report" model="ir.ui.view">
+            <field name="name">account.analytic.account.report</field>
+            <field name="model">account.analytic.account</field>
+            <field name="groups_id" eval="[(4, ref('analytic.group_analytic_accounting'))]"/>
+            <field name="field_parent">child_ids</field>
+            <field name="arch" type="xml">
+                <tree string="Analytic Accounts" groups="analytic.group_analytic_accounting">
+                    <field name="display_name"/>
+                    <field name="code"/>
+                    <field name="partner_id"/>
+                    <field name="account_type" invisible="1"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                    <field name="debit"/>
+                    <field name="credit"/>
+                    <field name="balance"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="action_analytic_account_report"
+                model="ir.actions.act_window">
+            <field name="name">Analytic Accounts</field>
+            <field name="res_model">account.analytic.account</field>
+            <field name="view_type">tree</field>
+            <field name="view_mode">tree,form</field>
+            <field name="search_view_id"
+                   ref="analytic.view_account_analytic_account_search"/>
+            <field name="context">{'search_default_active':1}</field>
+            <field name="view_id" ref="view_account_analytic_account_report"/>
+            <field name="domain">[('parent_id','=',False)]</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to add an analytic account.
+              </p>
+            </field>
+        </record>
+
+        <menuitem name="Analytic Accounts"
```

Comment:
I think the group must be in this menuitem, I still can see the analytic account hierarchy without analytic account enabled.  :eyes: